app/controllers/cluster_infos_controller.rb

```diff
@@ -1,7 +1,90 @@
 class ClusterInfosController < ApplicationController
   before_filter :load_experiment, only: :show
   include Scalarm::ServiceCore::ParameterValidation
-  #Id -> experimentId; simulations => 2,4,6,1,7
+
+=begin
+apiDoc:
+  @api {get} /cluster_infos/:id Cluster description
+  @apiName cluster_infos#show
+  @apiGroup ClusterInfos
+  @apiDescription When called, it validates 'simulations' parameter and then create list of integer values from this variable.
+  Next, it initialize ClusterInfos class and invoke 'evaluate' class method which firstly create result set and then calculate statistical factors.
+  Returns html or json with statistical information about simulation runs for input and output parameters.
+  Json contain calculated mean, variance, skewness, kurtosis, median, standard_deviation, upper/lower quartiles, argument ranges
+
+  @apiParam {String} id ID of experiment
+  @apiParam {String} simulations ids of simulation runs separated with comma
+  @apiParam {String} cluster_id ID (name) of cluster
+  @apiParam {String} chart_name name of method
+
+  @apiSuccessExample {json} Success-Response
```

Comment:
@Andriu01 - nie dało by się tutaj albo wyżej dodać jakiegoś komentarza do tego przykładowego JSON-a? Chodzi o to, że w data kluczami są kolejne statystyki, które są obiektami, w których każdy parametr wejściowy i wyjściowy jest kluczem. Przynajmniej gdzieś napisać, że tutaj parametry to "parameter1" i "parameter2", a MoEsy to "product" i "product_2".